src/cli/backup/groups_e2e_test.go

```diff
@@ -0,0 +1,615 @@
+package backup_test
```

Comment:
These files were added in https://github.com/alcionai/corso/pull/4101/files.  I'm not sure how they got sliced out.